src/go/rpk/pkg/cli/cmd/topic/consume.go

```diff
@@ -955,9 +975,14 @@ until one succeeds:
     9 digits              parsed as a unix second
     YYYY-MM-DD            parsed as a day, UTC
     YYYY-MM-DDTHH:MM:SSZ  parsed as RFC3339, UTC; fractional seconds optional (.MMM)
-    -dur                  duration ago; from now (as t1) or from t1 (as t2)
-    dur                   for t2 in @t1:t2, relative duration from t1
     end                   for t2 in @t1:t2, the current end of the partition
+    -dur                  a negative duration from now or from a timestamp
+    dur                   a positive duration from now or from a timestamp
+
+Durations can be relative to the current time or relative to a timestamp.
+If a duration is used for t1, that duration is relative to now.
+If a duration is used for t2, if t1 is a timestamp, then t2 is relative to t1.
+If a duration is used for t2, if t1 is a duration, then t2 is relative to now.
```

Comment:
What happens if t2 < t1? will it succeed or err?

Reply:
exits immediately because there are no offsets between a negative time range